src/i18n/en.json

```diff
@@ -1,414 +1,406 @@
 {
-    "home": "Home",
-    "wallet": "Wallet",
-    "domain.all": "All Teams",
-    "domain.root": "Root",
-    "role.disabled": "Disabled",
-    "role.member": "Member",
-    "role.admin": "Admin",
-    "role.founder": "Founder",
-    "role.0": "Recovery",
-    "role.1": "Root",
-    "role.2": "Arbitration",
-    "role.3": "Architecture",
-    "role.4": "Architecture Subdomain",
-    "role.5": "Funding",
-    "role.6": "Administration",
-    "role.unknown": "Unknown",
-    "button.accept": "Accept",
-    "button.approve": "Approve",
-    "button.cancel": "Cancel",
-    "button.choose": "Choose",
-    "button.decline": "Decline",
-    "button.delete": "Delete",
-    "button.dismiss": "Dismiss",
-    "button.edit": "Edit",
-    "button.remove": "Remove",
-    "button.retry": "Retry",
-    "button.submit": "Submit",
-    "button.ok": "Okay",
-    "button.yes": "Yes",
-    "button.no": "No",
-    "button.add": "Add",
-    "button.back": "Back",
-    "button.close": "Close",
-    "button.confirm": "Confirm",
-    "button.continue": "Continue",
-    "button.createMotion": "Create Motion",
-    "button.next": "Next",
-    "button.save": "Save",
-    "button.modify": "Modify",
-    "button.preview": "Preview",
-    "button.publish": "Publish",
-    "button.update": "Update",
-    "button.upgrade": "Upgrade",
-    "input.resetText": "Cancel",
-    "label.amount": "Amount",
-    "label.name": "Name",
-    "label.description": "Description",
-    "label.optional": "Optional",
-    "label.email": "Email address",
-    "label.password": "Password",
-    "label.force": "Force",
-    "label.banned": "Banned",
-    "label.loading": "Loading",
-    "label.waiting": "Waiting",
-    "placeholder.email": "handle@domain.com",
-    "placeholder.password": "Password",
+  "home": "Home",
```

Comment:
So I think the json were set at 4 spaces by prettier.

I don't mind the change to 2, but please make sure that the rule is enforced globally, otherwise, when we pull the updated file. our linters will (potentially) change it back

Reply:
Actually, it's a tricky one. There's an `.editorconfig` which specifies json files indentation as 4 spaces but there are a few files (`package.json`, `tsconfig.{dev,prod}.json`) that do not adhere to this which makes me think it's not effectively enforced.
Let's touch this tomorrow and I can address it separately. For now I have reverted the indentation changes I made.